packages/frontend/src/pages/AuthPopupResult.tsx

```diff
@@ -48,4 +48,36 @@ const AuthPopupResult: FC = () => {
     );
 };
 
+/**
+ * Fixed version of AuthPopupResult that is used in Github authentication
+ */
+export const SuccessAuthPopupResult: FC = () => {
```

Comment:
AuthPopupResult depends on a specific url, we only want to show a static success message and close the popup